includes/admin.form.inc

```diff
@@ -92,5 +92,49 @@ function islandora_entities_settings($form, &$form_state) {
     '#required' => TRUE,
   );
 
+  $form['islandora_entities_title_collection'] = array(
```

Comment:
These aren't actually referencing collections but instead Solr fields used for autocomplete, variable names such as `islandora_entities_scholar_title_solr_field` probably makes sense.